pkgs/by-name/re/rescrobbled/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, rustPlatform, pkg-config, openssl, dbus }:
+
+rustPlatform.buildRustPackage rec {
+
+  pname = "rescrobbled";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "InputUsername";
+    repo = "rescrobbled";
+    rev = "v${version}";
+    hash = "sha256-1E+SeKjHCah+IFn2QLAyyv7jgEcZ1gtkh8iHgiVBuz4=";
+  };
+
+  cargoLock.lockFile = src + "/Cargo.lock";
```

Comment:
The evaluation fails because `buildRustPackage` tries to read `lockFile` but since it is inside a derivation, it requires [import from derivation](https://nixos.org/manual/nix/stable/language/import-from-derivation.html), which is not allowed in Nixpkgs.